pylint/checkers/refactoring.py

```diff
@@ -261,13 +265,30 @@ def _check_redefined_argument_from_local(self, name_node):
                                  args=(name_node.name, ))
 
     @utils.check_messages('redefined-argument-from-local',
-                          'too-many-nested-blocks')
+                          'too-many-nested-blocks',
+                          'mutable-sequence-modified-in-loop')
     def visit_for(self, node):
         self._check_nested_blocks(node)
 
         for name in node.target.nodes_of_class(astroid.AssignName):
             self._check_redefined_argument_from_local(name)
 
+        inferrend = utils.safe_infer(node.iter)
+        if (isinstance(node.iter, astroid.Name) and
+                (isinstance(inferrend, type(astroid.util.Uninferable)) or
+                 isinstance(inferrend, astroid.node_classes.List)) and
+                node.body):
+            for part in node.nodes_of_class(astroid.Expr):
+                if (isinstance(part.value, astroid.Call) and
+                        isinstance(part.value.func,
```

Comment:
we have `is_method_call` in checkers/logging.py, seems like it may be reused (at least in some form) in this loop body.